behave/features/10.e2e_journey_tier_3_manual_address.feature

```diff
@@ -1,5 +1,5 @@
-@e2e_happy_path_no_nhs_login
-Feature: COVID-19 Shielded vulnerable people service - basic e2e user journey - no NHS login
+@feature_postcode_tier
```

Comment:
I suspect I don't fully understand this scenario, journey differences, and split between tags, but why is this not _core_?

Reply:
this isn't a core test its part of the feature work, its a test for the journey when the alert its 3 aka very high.